CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.5.2] - 2020-02-24
+
+### Changed
+- Bump authn-k8s client to v0.16.1 (cyberark/conjur-authn-k8s-client#70)
+
+### Fixed
+- Log into RH container registry before pushing (#1149)
```

Comment:
This probably should be changed to indicate that this fixes the push rather than implying that we fixed the logging in